src/routes.js

```diff
@@ -372,7 +372,8 @@ module.exports.bind = function (app) {
 
   account.get(paymentLinks.manage.index, permission('transactions:read'), paymentLinksController.getManage)
   account.get(paymentLinks.manage.disable, permission('tokens:create'), paymentLinksController.getDisable)
-  account.get(paymentLinks.manage.delete, permission('tokens:create'), paymentLinksController.getDelete)
+  account.get(paymentLinks.manage.delete, permission('tokens:create'), paymentLinksController.getDelete.get)
```

Comment:
Perhaps renaming getDelete to delete? getDelete.post is a bit contradictory

Reply:
I was avoiding renaming the file itself. But I agree with you. I will update it now. Thanks!

Reply:
Amended, thanks!